src/endpoints/transactions/entities/transaction.pool.ts

```diff
@@ -5,27 +5,27 @@ export class TransactionPool {
     Object.assign(this, init);
   }
 
-  @ApiProperty({ type: String, description: 'Transaction hash', example: '39098e005c9f53622e9c8a946f9141d7c29a5da3bc38e07e056b549fa017ae1b' })
+  @ApiProperty({ type: String, description: 'Transaction hash', example: '39098e005c9f53622e9c8a946f9141d7c29a5da3bc38e07e056b549fa017ae1b' , required: false })
```

Comment:
i think that for tx pool txHash, this field is required

Reply:
fixed